packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -1881,193 +1896,177 @@ describe('KeyringController', () => {
   describe('signTypedMessage', () => {
     describe('when the keyring for the given address supports signTypedMessage', () => {
       it('should throw when given invalid version', async () => {
-        await withController(
-          // @ts-expect-error QRKeyring is not yet compatible with Keyring type.
-          { keyringBuilders: [keyringBuilderFactory(QRKeyring)] },
```

Comment:
Some of these test scenarios were adding `QRKeyring` to builders, but without doing anything with it. These changes here may be a little confusing because of indentation change, but the only lines removed should be the ones related to `keyringBuilders`.